pkg/api/router.go

```diff
@@ -45,6 +45,18 @@ func (s *server) setupRouting() {
 		"POST": http.HandlerFunc(s.chunkUploadHandler),
 	})
 
+	router.Handle("/bzz-tag/name/{name}", jsonhttp.MethodHandler{
+		"GET": http.HandlerFunc(s.getANewTag),
```

Comment:
As this method changes the state, it should be `POST`. Also, maybe to rename it to `createTag`?

Reply:
This comment is left.